app/views/admin/delivery_partners/users/edit.html.erb

```diff
@@ -8,13 +8,13 @@
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">Edit user details</h1>
 
-    <%= form_with model: @delivery_partners_user_form, scope: :delivery_partners_user_form, url: admin_delivery_partners_user_path(@user) do |f| %>
```

Comment:
same here regarding missing the summary error box (not part of this pr)

Reply:
@cwrw 
Added, same for new page